light-system-programs/tests/provider.test.ts

```diff
@@ -76,11 +69,11 @@ describe("verifier_program", () => {
       2_000_000_000,
     );
 
-    RELAYER = new TestRelayer({
+    RELAYER = await new TestRelayer({
```

Comment:
we don't need the await for an object instantiation :)

Reply:
seeing that in several other places as well

Reply:
fixed ✅